libc/misc/resolve_path.c

```diff
@@ -475,6 +475,48 @@ TEST(resolve_path, symlink_rename)
 }
 
 
+/* check the behavior of the realpath function when it encounters pathname resolution of a symbolic link that results with a length longer than PATH_MAX */
```

Comment:
be specific and short with comments, eg.:

_check if path temporarily longer than PATH_MAX while resolving won't crash_

that way the if/else in the test makes sense (the implementation might support resolving the path or might return an error - crash is being considered a failure :))

Reply:
I've corrected it as Mateusz is not available (It could be good to merge it today to provide running this test on PR with new changes)